src/smart-components/App/stories/index.stories.js

```diff
@@ -216,6 +222,38 @@ export const user3 = () => fitPageSize(
   />
 );
 
+const CustomApp = () => {
+  const [channelUrl, setChannelUrl] = useState('');
+  return (
+    <Sendbird
+      appId={appId}
+      userId={array[3]}
+      nickname={array[3]}
+      theme="dark"
+      showSearchIcon
+      allowProfileEdit
+      profileUrl={addProfile}
+      imageCompression={{
+        compressionRate: 0.5,
+        resizingWidth: 100,
+        resizingHeight: '100px',
+      }}
+    >
+      <div style={{ display: 'flex', flexDirection: 'row', width: '100%', height: '100%' }}>
+        <ChannelList
+          onChannelSelect={(channel) => { setChannelUrl(channel.url) }}
+          queries={{ channelListQuery: { superChannelFilter: 'super' } }}
```

Comment:
```
const [channelListQuery, setChannelListQuery] = useState({ superChannelFilter: 'super' });
....
<ChannelList
          onChannelSelect={(channel) => { setChannelUrl(channel.url) }}
          queries={channelListQuery}
```
is better

Reply:
However, I don't need to set the query?

Reply:
`const [channelListQuery] = useState({ superChannelFilter: 'super' });`
Yeah this is better.. We might have used your original pattern before, but it refresh object inside `queries={{ channelListQuery: { superChannelFilter: 'super' } }}` every time, which causes rerenders